art_detectors/art_arcode_detector/launch/ar_code_detector_bundle.launch

```diff
@@ -6,12 +6,12 @@
 	<arg name="max_new_marker_error" default="0.08" />
 	<arg name="max_track_error" default="0.2" />
 
-	<arg name="cam_image_topic" default="/art/n2/kinect2/hd/image_color" />
-	<arg name="cam_info_topic" default="/art/n2/kinect2/hd/camera_info" />
+	<arg name="cam_image_topic" default="kinect2/hd/image_color" />
+	<arg name="cam_info_topic" default="kinect2/hd/camera_info" />
 
-	<arg name="output_frame" default="/marker" />
+	<arg name="output_frame" default="/$(arg machine)_kinect2_link" />
```

Comment:
tohle si myslím že nebude fungovat.. detektor počítá s tím že to příjde už v /marker...